### ISSUE TITLE
Decode error resulting from incorrect TypeStore maintenance

### ISSUE THREAD

**Issue by mdavidsaver** [mdavidsaver/pvxs#32]
https://epics.anl.gov/core-talk/2022/msg00526.php

```
2022-12-14T13:14:03.905922950 CRIT pvxs.tcp.io Server Error while processing cmd 0x0a: vector::_M_range_check 
```

**Comment by mdavidsaver** [comment]
The [packet capture](https://github.com/mdavidsaver/pvxs/files/10231448/pva-2022-12-14-1038.pcapng.gz) provided by @tboegi shows that the failed GET is retried successfully with a new TCP connection.  This is a sure sign of TypeStore inconsistency.

**Comment by mdavidsaver** [comment]
Should be fixed by c7b4650ba132b731f2dd1aff5b82dead6657fc88, uploaded as https://pypi.org/project/pvxslibs/1.1.1a1/
